closed/custom/Main-pre.gmk

```diff
@@ -0,0 +1,42 @@
+# ===========================================================================
+# (c) Copyright IBM Corp. 2020, 2020 All Rights Reserved
+# ===========================================================================
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.
+#
+# IBM designates this particular file as subject to the "Classpath" exception
+# as provided by IBM in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+# ============================================================================
+
+COMPONENTS_MAKE := $(MAKE) $(MAKE_ARGS) -f $(TOPDIR)/closed/Components.gmk
+
+ifeq (true,$(BUILD_JGSKIT))
+
+.PHONY : clean-openjceplus openjceplus-libs
+
+clean : clean-openjceplus
+
+clean-openjceplus :
+	@$(ECHO) Cleaning OpenJCEPlus native
+	@$(RM) -f $(addprefix $(OPENJCEPLUS_TOPDIR)/src/main/native/, com_ibm_crypto_plus_provider_icc_NativeInterface.h com_ibm_crypto_plus_provider_icc_FastJNIBuffer.h libjgskit.x)
+	$(MAKE) -C $(OPENJCEPLUS_TOPDIR)/src/main/native/ -f jgskit.mak cleanAll
+
+java.base-copy : openjceplus-libs
+
+openjceplus-libs :
+	@$(ECHO) Compiling OpenJCEPlus native
+	export PLATFORM=x86-linux64 JAVA_HOME=$(BOOT_JDK) GSKIT_HOME=$(GSKIT_HOME) JCE_CLASSPATH=$(OPENJCEPLUS_TOPDIR)/src/main/java/:$(JDK_OUTPUTDIR)/modules/java.base/ \
+		&& $(MAKE) -j1 -C $(OPENJCEPLUS_TOPDIR)/src/main/native/ -f jgskit.mak all
```

Comment:
Why is `-j1` required here?
Why does it seem to assume `x86-linux64`; are other platforms not supported?